bcmi/src/app/header/header.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Api } from '@services/api';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+
+export class HeaderComponent implements OnInit {
+  @Input() navigation: any;
+  hostname: string;
+  constructor( private api: Api) { }
+
+  ngOnInit() {
+    this.hostname = this.api.hostnameNRPTI;
+    window.scrollTo(0, 0);
```

Comment:
Same note as above, although I can see why this might be necessary bc it's the header.